app/examples/eigen_test.cpp

```diff
@@ -4,7 +4,7 @@
 // This is all done using the package Eigen.
 
 #include <iostream>
-#include <eigen3/Eigen/Dense>
+#include <Eigen/Dense>
```

Comment:
When importing Eigen using its CMake target, the include directory. is set to `<full-path>/include/eigen3`, so you just need to leave off the `eigen3` prefix. You are right to be confused, this does break with convention.